### ISSUE TITLE
Manage attendees

### ISSUE THREAD

**Issue by stephlocke** [lockedata/rcms#32]
## Detailed task
- Monitor sales
- Modify a registration e.g. issue a refund
- Send an email to attendees

## Assessing the task
Try to perform the task. Use google and the system documentation to help - part of what we're trying to assess how easy it is for people to work out how to do tasks.

Use a 👍 (`:+1:`) reaction to this task if you were able to perform the task. Use a 👎 (`:-1:`) reaction to the task if you could not complete it. Add a reply with any comments or feedback.

## Extra Info
- Site: [oconf](https://mysterious-coast-84721.herokuapp.com)
- System documentation: [ocw docs](http://openconferenceware.org/)
- Role: Conference team
- Area: Processes

**Comment by Amymcdougall** [comment]
@julierennes

**Comment by tuxette** [comment]
I don't think that registration is covered by this program. I've searched and did not find any place where tickets can be configured and sold. The documentation says:

- Anyone can list events
- Anyone can list/show sessions for an event
- Anyone can list/show proposals for an event
- Anyone can leave private comments about proposals to organizers
- Anyone can be informed of new proposals via ATOM feed
- Anyone can list/show tracks for an event
- Anyone can list/show session types for an event
- Anyone can subscribe to a feed of proposals for the event
- Anyone can list/show rooms for an event
- Anyone can create a profile, including a biography, picture and URLs
- Users can login via OpenID
- Users can create proposals until a deadline
- Users can update/delete their own proposals until a deadline
- Users can assign one or many speakers to a proposal
- Users can mark proposals/sessions as favorites

**Comment by Amymcdougall** [comment]
agree with [@tuxette](https://github.com/tuxette), I couldn't find this and read the same in the documentation.
